java-client/src/main/java/org/opensearch/client/opensearch/_types/analysis/LanguageAnalyzer.java

```diff
@@ -68,7 +69,7 @@ private LanguageAnalyzer(Builder builder) {
 
         this.version = builder.version;
         this.language = ApiTypeHelper.requireNonNull(builder.language, this, "language");
-        this.stemExclusion = ApiTypeHelper.unmodifiableRequired(builder.stemExclusion, this, "stemExclusion");
+        this.stemExclusion = builder.stemExclusion; // ApiTypeHelper.unmodifiableRequired(builder.stemExclusion, this, "stemExclusion");
```

Comment:
this should not be mandatory

Reply:
It is supported by most analyzers but not all

Reply:
but it is reasonable to make it optional for those that supports this property, I agree